ansible/roles/debops.postfix_virtual_ldap/defaults/main.yml

```diff
@@ -0,0 +1,685 @@
+
+                                                                   # ]]]
+# .. envvar:: postfix_virtual_ldap__ldap_start_tls [[[
+#
+# If ``True``, Postfix will use STARTTLS extension to make encrypted
+# connections to the LDAP server.
+postfix_virtual_ldap__ldap_start_tls: '{{ ansible_local.ldap.start_tls
+                                          if (ansible_local|d() and ansible_local.ldap|d() and
+                                              (ansible_local.ldap.start_tls|d())|bool)
+                                          else True }}'
+
+                                                                   # ]]]
+                                                                   # ]]]
+# LDAP settings [[[
+# ^^^^^^^^^^^^^^^^^
+
+# .. envvar:: postfix_virtual_ldap__ldap_user_filter [[[
+#
+# The LDAP filter used to look up user accounts in the directory.
+postfix_virtual_ldap__ldap_user_filter: '(&
+                                           (objectClass=inetOrgPerson)
+                                           (objectClass=postfixUser)
+                                         )
+                                         (mailEnabled=TRUE)
+                                         (|
+                                           (authorizedService=postfix)
+                                           (authorizedService=mail)
+                                           (authorizedService=\*)
+                                         )'
+
+                                                                   # ]]]
+# .. envvar:: postfix_virtual_ldap__ldap_domain_filter [[[
+#
+# The LDAP filter used to look up user accounts in the directory.
+postfix_virtual_ldap__ldap_domain_filter: '(&
+                                            (ObjectClass=dNSDomain)
+                                            (dc=%s)
+                                           )'
+
+                                                                   # ]]]
+# .. envvar:: postfix_virtual_ldap__ldap_quota_attribute [[[
+#
+# Set the LDAP attribute value to be read by Postfix in order to get the user quota.
+postfix_virtual_ldap__ldap_quota_attribute: 'mailQuota'
+                                                                   # ]]]
+# .. envvar:: postfix_virtual_ldap__ldap_quota_default [[[
+#
+# User default LDAP quota. Use human-readable values, e.g. "2 GB".
+postfix_virtual_ldap__ldap_quota_default: '10 GB'
+                                                                   # ]]]
+# .. envvar:: postfix_virtual_ldap__ldap_default_config [[[
+#
+# The LDAP configuration options defined by default.
+postfix_virtual_ldap__ldap_default_config: []
+
+                                                                   # ]]]
+# .. envvar:: postfix_virtual_ldap__ldap_config [[[
+#
+# List of custom LDAP configuration options defined for all hosts
+# in the Ansible inventory.
+postfix_virtual_ldap__ldap_config: []
```

Comment:
You most likely don't need the `postfix_virtual_ldap__*_ldap_config` variables.

Reply:
it doesn't bother here. But if you insist I will remove the vars.

Reply:
I think this type of config helps to keep the roles consistent with the host, groups, all inventory config

Reply:
I currently don't see `postfix_virtual_ldap__ldap_combined_config` used anywhere, do you have any plans to use it?

Reply:
When using different mail servers and domains changing the LDAP task this way is useful. But no, currently not used